code/modules/assembly/signaler.dm

```diff
@@ -63,7 +63,7 @@
 /obj/item/device/assembly/signaler/activate()
 	if(cooldown > 0)
 		return 0
-	cooldown = 2
+	cooldown = 0.5
```

Comment:
If it's actually one second, why not change it to 1 here? Anyone reading this code would think it was a bug.

Reply:
Fixed